triage.py

```diff
@@ -160,6 +160,10 @@ def is_mergeable(self):
             time.sleep(1)
         return self.instance.mergeable_state != "dirty"
 
+    def is_a_wip(self):
+        """Return True if PR has WIP in title"""
+        return "WIP" in self.instance.title or "[WIP]" in self.instance.title
```

Comment:
The exact implementation in Gitlab is that the title should *start* with "WIP: " or "[WIP] ". Although it seems with newer Gitlab releases they only hint into using "WIP: ". So this could be reduced to:

```python
return self.instance.title.startswith('WIP: ') or self.instance.title.startswith('[WIP] ')
```
PS Or you could drop the trailing whitespace if you want to be less pedantic than me :)